File: app/src/main/java/cn/intret/app/picgo/model/event/ConflictResolveResultMessage.java
package cn.intret.app.picgo.model.event;


import java.util.List;

import cn.intret.app.picgo.model.CompareItemResolveResult;

public class ConflictResolveResultMessage {
    List<CompareItemResolveResult> compareItems;

    public List<CompareItemResolveResult> getCompareItems() {
        return compareItems;
    }

    public ConflictResolveResultMessage setCompareItems(List<CompareItemResolveResult> compareItems) {
        this.compareItems = compareItems;
        return this;
    }
}

File: app/src/main/java/cn/intret/app/picgo/model/event/ConflictResolveResultMessage.kt
package cn.intret.app.picgo.model.event


import cn.intret.app.picgo.model.image.data.CompareItemResolveResult

class ConflictResolveResultMessage {
    internal lateinit var compareItems: List<CompareItemResolveResult>

    fun getCompareItems(): List<CompareItemResolveResult> {
        return compareItems
    }

    fun setCompareItems(compareItems: List<CompareItemResolveResult>): ConflictResolveResultMessage {
        this.compareItems = compareItems
        return this
    }
}
